src/main/resources/META-INF/neoforge.mods.toml

```diff
@@ -15,28 +15,28 @@ logoFile = "assets/${modId}/icon.png"
 [properties]
 "connector:placeholder" = true
 
-[[dependencies.${ modId }]]
```

Comment:
unclear why this would be the case - is this a change in neo?

Reply:
intellij idea throws a warning when the spaces are present. the example neoforge.mods.toml omits them as well